install/uninstall.ps1

```diff
@@ -0,0 +1,95 @@
+# Enable a default -Verbose flag for debug output.
+[CmdletBinding()]
+
+# Halt script execution at the first failed command.
+$script:ErrorActionPreference='Stop'
+
+$deleted = $false
+
+function Uninstall {
+    $uninstallDirectory = Get-UninstallDirectory
+    Remove-InstallationFolder $uninstallDirectory
+    Remove-FromPath $uninstallDirectory
+
+    if ($deleted) {
+        Write-Output "Uninstalled AzureAuth!"
+    } else {
+        Write-Output "There were no AzureAuth installations to be deleted."
+    }
+}
+
+function Get-UninstallDirectory {
+    $directory = if (![string]::IsNullOrEmpty($Env:AZUREAUTH_INSTALL_DIRECTORY)) {

```

Comment:
What if the env `AZUREAUTH_INSTALL_DIRECTORY` refers to the `%localappdata%/Programs/AzureAuth`?

Reply:
Good point, I'll make a check for that.

Reply:
Definitely can edge case we handle. However, our [doc ](https://github.com/AzureAD/microsoft-authentication-cli/blob/c1296b3d9777f8ace89e3621de0d433275252d98/README.md#windows)clearly specifies that we install at `%localappdata%/Programs/AzureAuth` by default unless the env var is set. So if someone sets that env var to default location, that would be weird, but worth handling. :)

Reply:
True that!

Reply:
Added the check for it.